src/frontend/query_protocol.ml

```diff
@@ -96,6 +96,13 @@ type error_filter = {
   typing : bool;
 }
 
+type info = 
```

Comment:
Since there is a lot of other things defined in that module that are unrelated to the `Syntax_document` command we should use a more descriptive name like `syntax_info` or `syntax_document_result`.

One way to see that this name is not precise enough is is when you use it elsewhere like in ocaml-lsp: `Query_protocol.info` is really non-informative :-)